content/content-strategy/set-up-governance/topic-takeaway.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: text/topictakeaway
+heading: Content strategy output
+firstcolumnheader: What you'll get
```

Comment:
I would split these by `previous` and `next`. First column and second column can be confusing for users entering in content.

Reply:
Roger!

Reply:
Resolved